add "back to top" links

sfarajnia opened the pull request:
Tables of contents are well written and structured the notebooks in a nice manner.
However, the content of the notebook would be more user-friendly if there would be "back to top" link beside the titles inside the notebook.
I've added "back-to-top" links to python notebook (Basics, Object Oriented Programming and Modular Programming)

Commit:
updated python material

Commit:
updated python material

Commit:
updated python material

Commit:
updated basics with new TOC

Commit:
updated python with new TOC

Commit:
renamed python

Commit:
renamed python

Commit:
add logos

Commit:
added images

Commit:
update all notebooks with logos

Commit:
add more logos

Commit:
delete unnecessary files

Commit:
update README

Commit:
remove extra TOC

Commit:
add banners to python tutorials

Commit:
fix for decorators session

Commit:
update TOC

hejazizo committed:
Set theme jekyll-theme-minimal

MahtabRanjbar committed:
Updating The meaning of Underscore Tutorial

Commit:
add CONTRIBUTING instrcutions

hejazizo committed:
Merge pull request #2 from MahtabRanjbar/main

Updating The meaning of Underscore Tutorial

Commit:
update single underscore notebook

Commit:
add how to contribute

MahtabRanjbar committed:
Update pathlib tutorial

hejazizo committed:
Merge pull request #3 from hejazizo/update-tutorial

Update pathlib tutorial

Commit:
update sessions style

Commit:
update pathlib session titles

sfarajnia committed:
add "back to top" links

tables of contents are well written and structured the notebooks in a nice manner.
The content of the notebook would be more user_friendly if there would be "back to top" link beside the titles inside the notebook.

Commit:
add concurrency sessions

MahtabRanjbar commented:
Writing "Back to top" for each subtopic does not seem very nice. I think it is better to use the relevant icon instead of writing "back to top".

sfarajnia commented:
> Writing "Back to top" for each subtopic does not seem very nice. I think it is better to use the relevant icon instead of writing "back to top".

Good idea! I will replace text with an icon.

sfarajnia committed:
Merge branch 'main' into salman_modify_links

Commit:
add concurrency sessions

Commit:
fix image path

sfarajnia committed:
add back_to_top ICONS

sfarajnia committed:
Merge branch 'main' of https://github.com/hejazizo/CS-Tutorial into main

sfarajnia committed:
Merge branch 'salman_modify_links' into main

sfarajnia committed:
updat back_to_top ICONS

sfarajnia commented:
> Writing "Back to top" for each subtopic does not seem very nice. I think it is better to use the relevant icon instead of writing "back to top".

hi, again. 
I've updated all notebooks and added ICONS for "back to top" links.
This is a screenshot from one of the titles of the advanced folder. The "blue top_arrow", if clicked, goes to the table of contents:  
![image](https://user-images.githubusercontent.com/10610881/145671551-875568d6-0738-4380-ac67-dd10f90b8e4a.png)

MahtabRanjbar commented:
@sfarajnia 
 good job. I think you also feel it has a nicer and cleaner structure now.

hejazizo commented:
After putting some thought into this, I think we should not do this manually.

You can simply write a code to add the link to a jupyter notebook file cause it's basically a json file.
You should add a cell at the end of it that contains a back to top link.

arf1372 commented:
> After putting some thought into this, I think we should not do this manually.

I don't get what your point is. Having a "back to top" link besides each heading would be nice-to-have (actually, I don't use them but someone may find it useful).

> You can simply write a code to add the link to a jupyter notebook file cause it's basically a json file. You should add a cell at the end of it that contains a back to top link.

Also, I don't understand what you mean by mentioning `ipynb` files are just jsons.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/pytopia/CS-Tutorial?pullRequest=4) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/pytopia/CS-Tutorial?pullRequest=4) before we can accept your contribution.<br/>**1** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: hejazizo<br/>:x: Ali<br/>:x: MahtabRanjbar<br/>:x: sfarajnia<br/><hr/>**Ali** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/pytopia/CS-Tutorial?pullRequest=4) it.</sub>
